### ISSUE TITLE
ubuntu 14.04

### ISSUE THREAD

**Issue by ninad458** [Genymobile/scrcpy#118]
meson x --buildtype release --strip -Db_lto=true
The Meson build system
Version: 0.46.0.dev1
Source dir: /home/ed/scrcpy
Build dir: /home/ed/scrcpy/x
Build type: native build
Project name: scrcpy
Native C compiler: cc (gcc 4.8.4 "cc (Ubuntu 4.8.4-2ubuntu1~14.04.4) 4.8.4")
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /usr/bin/pkg-config (0.26)

app/meson.build:21:0: ERROR:  Native dependency 'libavformat' not found

noob here. please eli5.

**Comment by rom1v** [comment]
> ERROR: Native dependency 'libavformat' not found

Did you [install](https://github.com/Genymobile/scrcpy#linux) `libavformat-dev`?

**Comment by ninad458** [comment]
>Did you install libavformat-dev?
about that:
sudo apt install make gcc pkg-config meson \
                 libavcodec-dev libavformat-dev libavutil-dev \
                 libsdl2-dev

gives me 
E: Unable to locate package meson

**Comment by rom1v** [comment]
Ubuntu 14.04 is very old, `meson` is probably not packaged. Instead, try to install `pip3` (or `pip`), then:

    pip3 install meson

**Comment by ninad458** [comment]
pip3 install meson
bash: /usr/lib/command-not-found: /usr/bin/python3: bad interpreter: No such file or directory
pip install meson
bash: /usr/lib/command-not-found: /usr/bin/python3: bad interpreter: No such file or directory

**Comment by Magnesus** [comment]
I get "meson version is 0.29.0 but project requires >= 0.37." onUbuntu 16.04. :(
